no-issue added entry for the global time context in a dashboard. (#1013)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/dashboards.md

You can only add widgets and data to the dashboard for the device itself. It is not possible to add data from child devices because the structure of these devices might be different from device to device.
{{< /c8y-admon-info >}}

<a name="global-time-context"></a>
### Global time context

> Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the global time for the entire dashboard or particular widgets. This ensures that all data presented in the dashboard is aligned with the same time context, regardless of the widget being used.

Reviewer: ```
Widget developers can set the global time for the entire dashboard or particular widgets.
```
user reading this may thought that we can turn on some switch on dashboard and select widgets that are bound to global date context... but the truth is that we configure widgets to use global date context and widget configuration causes time range selector to show on header bar.
So it's more like:
 'Widget developers can set widget config to use global date context (which will be created with first widget configured to use it) and this shared context can be used by other widgets that also has been configured to be bound to global date context.'

> Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the global time for the entire dashboard or particular widgets. This ensures that all data presented in the dashboard is aligned with the same time context, regardless of the widget being used.

Author: ```suggestion
Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set widget config to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.
```

> Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the global time for the entire dashboard or particular widgets. This ensures that all data presented in the dashboard is aligned with the same time context, regardless of the widget being used.

Author: ```suggestion
Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set widget config to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.
```

> Global time context a allows users to set a time range for their dashboards and align all data to that range. widget developers can set the global time for the entire dashboard or particular widgets. This ensures that all data presented in the dashboard is aligned with the same time context, regardless of the widget being used.

Author: ```suggestion
Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the global time for the entire dashboard or particular widgets. This ensures that all data presented in the dashboard is aligned with the same time context, regardless of the widget being used.
```

> Global time context allows users to set a time range for their dashboards and align all data to that range. Widget developers can set widget config to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.

Reviewer 2: ```suggestion
The global time context functionality allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the widget configuration to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.
```

Reviewer 2: @eniosultan Actually only the first sentence is of interest for users, the rest should go into the Web SDK docs, shouldn´t it? But we of course should somehow state that the functionality must be set in the widget config.

Author: I rephrased the second sentence accordingly.

> The global time context functionality allows users to set a time range for their dashboards and align all data to that range. Users can set the widget configuration to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.

Reviewer 2: Users can set the widget configuration to use global date context which will be created with the first widget configured to use it.
> I still do not get that sentence, it sounds strange :-). Where and how can users do that?

Author: In the widget configuration screen. See image below. 

![global-time-context](https://github.com/SoftwareAG/c8y-docs/assets/92023325/ed6ab393-6527-40f3-bce9-4710c0949d9a)

> The global time context functionality allows users to set a time range for their dashboards and align all data to that range. Users can set the widget configuration to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.

Reviewer 2: ```suggestion
In the widget configuration, you can specify a global time range for your dashboards. When using a global time range, all data in a dashboard will be aligned to this range. After setting the global time range in a first widget, this shared context can then be used by other widgets that are configured to use the global date context.
```

Reviewer 2: I rephrased it a bit :-)

> The global time context functionality allows users to set a time range for their dashboards and align all data to that range. Widget developers can set the widget configuration to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.

Author: ```suggestion
The global time context functionality allows users to set a time range for their dashboards and align all data to that range. Users can set the widget configuration to use global date context which will be created with the first widget configured to use it. This shared context can then be used by other widgets that also have been configured to be bound to the global date context.
```

> The global time range can be set to any duration, from the last hour to the last month, depending on the monitoring needs. The time range is displayed at the top of the dashboard, so you can quickly see the context in which the data is being presented. Additionally, you can share dashboards with a specific time context by including the time context parameters in a URL.

Reviewer: ```suggestion
The global time range can be set to any time span, from the last minute to the last month (or more, using custom dates picker), depending on the monitoring needs. The time range is displayed at the top of the dashboard, so you can quickly see the context in which the data is being presented. Additionally, you can share dashboards with a specific time context by including the time context parameters in a URL. URL parameters will be set automatically when user sets global date context with picker.
```

> The global time range can be set to any time span, from the last minute to the last month (or more, using custom dates picker), depending on the monitoring needs. The time range is displayed at the top of the dashboard, so you can quickly see the context in which the data is being presented. Additionally, you can share dashboards with a specific time context by including the time context parameters in a URL. URL parameters will be set automatically when user sets global date context with picker.

Reviewer 2: ```suggestion
The global time range can be set to any time span, depending on the monitoring needs, by using a custom date picker. The time range is displayed at the top of the dashboard, so you can immediately see the context for which the data is presented. Additionally, you can share dashboards with a specific time context by including the time context parameters in a URL. URL parameters will be set automatically when users set the global time context with the picker.
```

{{< c8y-admon-info >}}
This feature is currently available for the following widgets: Event list, Data points graph, and Data points table.
{{< /c8y-admon-info >}}

<img src="/images/users-guide/cockpit/cockpit-dashboard-global-time.png" name="Global time context"/>

> Each widget synchronized with the dashboard time range will feature an icon denoting this at the top right corner of the widget.

Reviewer 2: ```suggestion
Widgets synchronized with the dashboard time range are indicated by a respective icon at the top right of the widget.
```
